x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/action/XPackUsageFeatureAction.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+package org.elasticsearch.xpack.core.action;
+
+import org.elasticsearch.action.Action;
+import org.elasticsearch.xpack.core.XPackField;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class XPackUsageFeatureAction extends Action<XPackUsageFeatureResponse> {
```

Comment:
do you mind adding brief javadocs explaining that this action is shared and in core but the various implementations live in the other plugins?